src/PathItem.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const PathItem = ({pathName, pathItem}) => {
+  return (
+    <div>
+      <h2>{pathName}</h2>
+      {/* render Operation(s) */}
+    </div>
+  )
+};
+
+PathItem.propTypes = {
+  pathName: PropTypes.string.isRequired,
+  pathItem: PropTypes.object.isRequired
```

Comment:
I think we can go deeper on these and defined the object "shape." That has me thinking though, since many of these OpenAPI objects are reused throughout parts of the app should we define them in one place and import them? Not sure if there is any precedent for that floating around 🤷‍♂️